### ISSUE TITLE
Initiative roll should add or update to turn order

### ISSUE THREAD

**Issue by jazeee** [epic-power-rpg/roll20#73]
I tested in the Mods script area, and found that
```
    findObjs({_pageid: pageid,
        _type: 'graphic'});
```
returns an empty array. Due to this, the turn order is not updated.

Needs further investigation.
@johnlamping - this is what I found so far.

**Comment by johnlamping** [comment]
I thought it might have something to do with transmogrified characters, but
I'm also seeing the problem in a brand new character in Alanya.
I tried things in Carlonia, which still has the old character sheet, and
initiative is working right there. So it does not appear to be a roll20
change, but something about the new character sheet.

P.S. Here is how someone else showed how to get all tokens for a character.

findObjs({type:graphic}).filter((o)=>o.get('represents')===characterid);

**Comment by jazeee** [comment]
I wonder if it is related to the upgrade to non-legacy.
I'll play around with it...

**Comment by jazeee** [comment]
What we do is basically:
```
findObjs({_pageid: pageid, _type: 'graphic'}).filter((token) =>
        token.get('represents') == character_id)
```

The specific difference is the properties passed to `findObjs`
I'll see what happens if we remove the `_`

**Comment by jazeee** [comment]
I went back to https://github.com/epic-power-rpg/roll20/tree/5e16716a2b886b4ad78d959d7f194911fdef6909 which was the commit when you added the initial API_Scripts.js file.
Tested in JazLanya.
I still couldn't get the turn order to update even with the legacy UI and code.

I wonder if we should get the api code from Carlonya and compare.

**Comment by jazeee** [comment]
Ahh, ok, I got it working using legacy code on JazLanya.
I forgot to add the character on to the map.

**Comment by jazeee** [comment]
Works at 4b9a860ed4de0b455dc41a48ddf92b8f65e65480.
Doing a binary search of commits.

**Comment by jazeee** [comment]
Ok, the breakage occurred in my PR: https://github.com/epic-power-rpg/roll20/pull/20

**Comment by jazeee** [comment]
Found it... An extra `"` in the string. Fixin gnow

**Comment by jazeee** [comment]
Fixed in #74
